File: ano_bissexto.py
n = int(input())

if n % 4 != 0:
    print("NAOBISSEXTO")
else:
    if n % 100 != 0:
        print("BISSEXTO")
    else:
        if n % 400 != 0:
            print("NAOBISSEXTO")
        else:
            print("BISSEXTO")

File: ano_bissexto.cpp
#include <iostream>

int main() {
    int year;
    std::cin>>year;
    if (year % 4 != 0 or(year%100==0 and year%400!=0)){
        std::cout<<"NAOBISSEXTO";
    }else{
        std::cout<<"BISSEXTO";
  }
}